src/implementations/twig/components/footer-standardised/footer-standardised.html.twig

```diff
@@ -49,103 +47,94 @@
+        {% endif %}
+        <div class="{{ _section_class_name }}">
+          {# logo (EU) #}
+          {% if _sections.logo is defined and _sections.logo is not empty %}
+            {% set _label %}
+              <img
+                alt="{{ _sections.logo.alt | e('html_attr') }}"
+                title="{{ _sections.logo.title }}"
+                class="ecl-footer-standardised__logo-image-mobile"
+                src="{{ _sections.logo.src_mobile }}"
+              />
+              <img
+                alt="{{ _sections.logo.alt | e('html_attr') }}"
+                title="{{ _sections.logo.title }}"
+                class="ecl-footer-standardised__logo-image-desktop"
+                src="{{ _sections.logo.src_desktop }}"
+              />
+            {% endset %}
+            {% include '@ecl/link/link.html.twig' with {
+              link: {
+                path: _sections.logo.path,
+                label: _label,
+                type: 'standalone',
+                aria_label: _sections.logo.title
+              },
+              extra_classes: 'ecl-footer-standardised__logo-link'
+            } only %}
+          {% endif %}
+          {# content before #}
+          {% if _sections.content_before is defined and _sections.content_before is not empty %}
+            <div class="ecl-footer-standardised__content">{{ _sections.content_before }}</div>
+          {% endif %}
+          {# title #}
+          {% if _sections.title is defined and _sections.title is not empty %}
+            {% set _title_class_name = 'ecl-footer-standardised__title' %}
+            {% if _sections.title_with_separator is defined and _sections.title_with_separator %}
+              {% set _title_class_name = _title_class_name ~ ' ecl-footer-standardised__title--separator' %}
+            {% endif %}
+            {% if _sections.title.link is defined and _sections.title.link is not empty %}
+              {% include '@ecl/link/link.html.twig' with _sections|merge({
+                link: _sections.title.link|merge({ type: 'standalone'}),
+                extra_classes: _title_class_name,
+                icon: _sections.title.icon|default({}),
+              }) only %}
+            {% else %}
+              <div class="{{ _title_class_name }}">
+                {{- _sections.title -}}
+              </div>
+            {% endif %}
+          {% endif %}
+          {# description #}
+          {% if _sections.description is defined and _sections.description is not empty %}
+            <div class="ecl-footer-standardised__description">{{ _sections.description }}</div>
+          {% endif %}
+          {# list #}
+          {% if _sections.links is defined and _sections.links is not empty and _sections.links is iterable %}
+            {% set _list_class_name = 'ecl-footer-standardised__list' %}
+            {% if _sections.links_inline is defined and _sections.links_inline %}
```

Comment:
As before, it's a boolean, if it's false it would not pass: `{% if _sections.links_inline %}`